config/routes.rb

```diff
@@ -1,12 +1,13 @@
 Rails.application.routes.draw do
 
-  root 'questions#index'
+	root 'questions#index'
 
-  resources :questions do
-    resources :answers
-  end
+	resources :questions do
+		resources :answers
+		resources :categories
```

Comment:
@chrisseldo @I-love-ruby  do not merge any more @gabalicious  commits until he fixes this indentation mess he's making.  Looks unprofesh.